src/User.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+
+public class User {
+    private String userName;
+    private String action;
+    private double balance = 100.0;
+    private double withdraw;
+    private String cancel;
+    Scanner input = new Scanner(System.in);
+
+
+    public String getUser() {
+        return userName;
+    }
+    public String getAction() {
+        return action;
+    }
+    public double getBalance() {
+        return balance;
+    }
+    private double getWithdraw() {
+        return withdraw;
+    }
+    public String getCancel() {
+        return cancel;
+    }
+    public void setUser(String user)  {
+        this.userName = user;
+    }
+    public void setBalance(double balance) {
```

Comment:
As mentioned in class, architecturally I would probably not have a setter that takes an arbitrary amount as a new balance.  I would have 2 methods that handle debit and credit.